src/utils/style-formatter.js

```diff
@@ -77,17 +79,47 @@ export default {
 
     return styles;
   },
-  setFontSizeAndFamily({ button, Theme, layout }) {
+  createLabelAndIcon({ button, Theme, layout }) {
     const { style } = layout;
-    const text = button.firstElementChild;
-    text.textContent = style.label;
-    const fontFamily = colorUtils.getFontFamily(Theme);
-    setFontsize(text, button.clientHeight, fontFamily);
+    button.innerHTML = '';
+    const text = document.createElement('text');
+    text.style.whiteSpace = 'nowrap';
+
+    // textSpan containing the text. Only set text decoration here otherwise button will get underline as well
```

Comment:
@niekvanstaveren you mean the icon will be underlined? And we don't want that?

Reply:
Ah yes that is what I meant. I don't think we want that, because it looks very ugly (try it)